paddle/fluid/framework/executor.cc

```diff
@@ -466,6 +466,13 @@ void Executor::RunPartialPreparedContext(ExecutorPrepareContext* ctx,
 #else
       PADDLE_THROW(
           platform::errors::Unimplemented("No XPU gc found in CPU/GPU paddle"));
+#endif
+    } else if (platform::is_npu_place(place_)) {
+#ifdef PADDLE_WITH_ASCEND_CL
+
```

Comment:
Add LOG and TODO here if not implemented yet

Reply:
Thanks, done